packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -1342,7 +1342,9 @@ export const createAzureCommunicationCallWithChatAdapterFromClients: ({ callClie
 export const createAzureCommunicationChatAdapter: ({ endpoint: endpointUrl, userId, displayName, credential, threadId }: AzureCommunicationChatAdapterArgs) => Promise<ChatAdapter>;
 
 // @public
-export const createAzureCommunicationChatAdapterFromClient: (chatClient: StatefulChatClient, chatThreadClient: ChatThreadClient) => Promise<ChatAdapter>;
+export const createAzureCommunicationChatAdapterFromClient: (chatClient: StatefulChatClient, chatThreadClient: ChatThreadClient, options?: {
```

Comment:
If the feature is beta we need to avoid this change in the stable API (once it's in a stable release we can't change the name, and this will be reviewed by ARB for any stable release so without the accompanying feature being released we shouldn't expose it).

I assume conditionally compiling the options isn't working? (cc @PorterNan). If not we should be able to create an overload with the new param and only export the overload in beta

Reply:
We can create an overload for sure. Some of the feedback I got was to not create an overload cause this is an optional argument anyways and shouldn't be a breaking change. And on top of that this style follows similar to what the Call Adapter is already doing.

Reply:
Is it working for CC? If not, maybe it is a good sign we should support that for CC, I've seen something similar for 5+ times

Reply:
Here we go: 
https://github.com/Azure/communication-ui-library/pull/2880

Reply:
Will wait for the above PR to go in before merging this PR.